Mathlib/FieldTheory/Normal.lean

```diff
@@ -162,12 +162,11 @@ theorem AlgEquiv.transfer_normal (f : E ≃ₐ[F] E') : Normal F E ↔ Normal F
 -- salient in the future, or at least taking a closer look at the algebra instances it uses.
 attribute [-instance] AdjoinRoot.instSMulAdjoinRoot
 
-set_option maxHeartbeats 210000 in
+set_option maxHeartbeats 300000 in
 theorem Normal.of_isSplittingField (p : F[X]) [hFEp : IsSplittingField F E p] : Normal F E := by
   rcases eq_or_ne p 0 with (rfl | hp)
-  · have := hFEp.adjoin_roots
-    simp only [Polynomial.map_zero, roots_zero, Multiset.toFinset_zero, Finset.coe_empty,
-      Algebra.adjoin_empty] at this
+  · have := hFEp.adjoin_rootSet
+    simp only [rootSet_zero, Algebra.adjoin_empty] at this
```

Comment:
In https://github.com/leanprover-community/mathlib4/pull/4891 the proof was the longer
```lean
simp only [Polynomial.map_zero, roots_zero, Multiset.toFinset_zero, Finset.coe_empty, rootSet,
      Algebra.adjoin_empty] at this
```
Its weird that this proof is somehow faster.

Reply:
Since this works with `maxHeartbeats 210000` I think we can use it, but it's probably not very important.

In any case everything LGTM, thanks!

bors d+